src/test/init/features/storage.spec.ts

```diff
@@ -22,13 +23,30 @@ describe("storage", () => {
 
   describe("doSetup", () => {
     it("should set up the correct properties in the project", async () => {
-      const setup = { config: {}, rcfile: {} };
+      const setup = {
+        config: {},
+        rcfile: {},
+        projectId: "my-project-123",
+        projectLocation: "us-central",
+      };
       promptStub.returns("storage.rules");
       writeProjectFileStub.resolves();
 
       await doSetup(setup, new Config("/path/to/src", {}));
 
       expect(_.get(setup, "config.storage.rules")).to.deep.equal("storage.rules");
     });
+
+    it("should error when cloud resource location is not set", async () => {
+      const setup = {
+        config: {},
+        rcfile: {},
+        projectId: "my-project-123",
+      };
+
+      expect(doSetup(setup, new Config("/path/to/src", {}))).to.eventually.be.rejectedWith(
+        FirebaseError
```

Comment:
the second parameter of `rejectedWith` is a string or regex that is checked against the error message. Can that be provided to check for a more specific error?

Reply:
Done and added to Firestore tests as well. Thanks!